tests/phpunit/tests/theme/wpEnqueueStoredStyles.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+require_once __DIR__ . '/base.php';
+
+/**
+ * Tests wp_enqueue_stored_styles().
+ *
+ * @group dependencies
+ * @group scripts
```

Comment:
I don't think these groups are needed. While the function lives in the `script-loader.php` and uses Styles from the Dependencies API, this function is for the themes group. Removing these 2 groups from the commit.